rs/p2p/quic_transport/src/connection_manager.rs

```diff
@@ -313,7 +322,14 @@ impl ConnectionManager {
                 },
                 Some(conn_res) = self.inbound_connecting.join_next() => {
                     match conn_res {
-                        Ok(conn_out) => self.handle_connecting_result(conn_out, None),
+                        Ok(Ok(conn)) => self.handle_established_connection(conn.connection, conn.peer_id),
+                        Ok(Err(err)) => {
+                            self.metrics
+                                .connection_results_total
+                                .with_label_values(&[CONNECTION_RESULT_FAILED_LABEL])
```

Comment:
I see we already log it, but would it also be a good idea to have different metrics for failed inbound and outbound connections?

Reply:
Maybe, I am yet to figure out what is the best logs and metrics. I would prefer to keep it as is, since it is not change the old metric.